index.html

```diff
@@ -4674,7 +4674,7 @@ <h2>Internationalization Considerations</h2>
 standards or protocols implementation, ignoring internationalization makes it
 difficult for data to be produced and consumed across a disparate set of
 languages and societies, which would limit the applicability of the
-specification and significantly diminish its value as a standard.
+specification and significantly diminish its value as a global standard.
```

Comment:
It's a personal nit, but I tend to omit qualifying adjectives like "global" and "internationalized" (you'll see another comment later to this effect) unless they have some additional meaning because it implies that there is an alternative.

Reply:
Fixed in b2f3fdb952a9ca4a96411170bb5d465334cdcad3.